docs/dev/table/connectors/blackhole.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Blackhole SQL Connector"
+nav-title: Blackhole
+nav-parent_id: sql-connectors
+nav-pos: 6
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<span class="label label-primary">Sink: Bounded</span>
+<span class="label label-primary">Sink: UnBounded</span>
+
+* This will be replaced by the TOC
+{:toc}
+
+The Blackhole connector allows for swallowing all input records. It is designed for:
+
+- high performance testing.
+- UDF to output, not substantive sink.
+
+Just like /dev/null device on Unix-like operating systems.
+
+The Print connector is built-in.
+
+How to create an Blackhole table
+----------------
+
+Although it doesn't make sense to define the fields of print table, you need to write them all in DDL.
```

Comment:
I understand what you are trying to say but the statement sounds like a put down on Flink. Lets just drop it.
```suggestion
```